src/includes/getting-started-install/javascript.angular.mdx

```diff
@@ -1,3 +1,11 @@
+<Alert level="info" title="Angular Version Compatibility" >
+
+The latest version of the Sentry Angular SDK officially supports Angular 10 and newer.
+If you are using an older Angular version and you experience problems with the SDK, you can
+try using version 6 (at your own risk) until you upgrade your Angular version.
```

Comment:
This isn't what we want to say right? If you are using an old angular version and you have problems, you **should** use v6 (which is not risky I believe). Or does v6 also not support angular version less than 10? If so we should clarify that.

Reply:
So here's the thing:

Like v7, v6 also has NG10-13 in its peer deps. So "officially" (but unfortunately nowhere specified, except for `package.json`) it also only supported these versions. If you wanted to use it with NG <10, you have to specify the `--force` flag when `npm install`ing. In that sense, it's "risky" but apart from that it seems to work for people (except for `TraceDirective`).

I want to put in writing that v7 officially supports 10-13. It _seems_ to support NG9 as well but I don't think we should officially add an older NG version to v7  when v6 didn't include it (also, NG9 is old).  Which is why I don't want to explicitly highlight the fact that that one can use v7 with NG9. At the same time I don't want to say "if you're using NG<10, use v6" because NG9 users could still upgrade... I know that's kinda a stupid situation. 

Therefore, I wanted say something like "if you're using v7 with older NG versions and experience problems, downgrade to v6". Does that kinda make sense? 
I can remove the "(at your own risk)". My intent here was to state that we're not gonna support v6 further. I can generally try to rewrite this to better reflect my intention.

Reply:
Hmm ok I get it now. I would change the wording of the first sentence to something along the lines of:

> The Sentry Angular SDK officially only supports Angular version 10 and newer.

Implying that even Sentry SDK v6 doesn't support Angular v9.

Then we could add additional info:

> If you need to use Angular version 9 and the latest version of the Sentry SDK gives you trouble, try downgrading the Sentry SDK to version 6. Version 6 of the Sentry SDK was compiled differently and might work with older versions of Angular. Please do this at your own risk, as this combination of packages is not being maintained and tested.

Wdyt? We can ofc still adapt the wording.

Reply:
Thanks for the suggestion. I added it (with some small adjustments) and repositioned the info box as well. Wdyt, does it sound better now?

Reply:
Reads well now! Thanks for considering my input :D